src/main/java/drug_request_frameworks_drivers/DrugRequestScreen.java

```diff
@@ -0,0 +1,22 @@
+package drug_request_frameworks_drivers;
+
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class DrugRequestScreen extends JPanel implements ActionListener {
+    /**
+     * The drugName requested by the site
+     */
+    JTextField drugName = new JTextField(15);
+
+    /**
+     * number of drugBottles requested by the site
+     */
+    J
```

Comment:
Is this J suppose to be here?

Reply:
Hey Joshua, Yes you're correct, I wasn't finished and should have left a comment instead. Thanks for catching that.